### DISCUSSION TITLE
Rom Box Art Separation

### DISCUSSION THREAD

**Discussion by sere83** [OnionUI/Onion#563] [Ideas]
Hey there, just a small suggestion. Do you think devs could implement a background and separator for the box art when scrolling through the roms menus?

Looks a bit odd with the album art just floating / obstructing the text, You could cut the rom titles characters and put the rom title text on scroll left to right when highlighted so you can read all the titles maybe?

Here is a basic mockup, could this work do you think?

![Untitled-2](https://user-images.githubusercontent.com/51299496/193365932-25e558ea-a23a-4553-9852-1cb58b18dadc.jpg)

**Comment by Aemiii91** [comment]
This is based on personal preference and depend on which box art style you use. This is achievable by changing the `skin/preview-bg.png` image in  your current theme. And to get the padding around the box art, you could make the box art images smaller than 250 pixels. The ellipsis (`...`) will not be possible, since we don't have access to the source.

**Comment by Aemiii91** [comment]
Maybe there is hope for implementing something like this in Tweaks theme overrides. The problem being doing something that fits with all themes. But I guess a black background will always fit.

**Reply by sere83** [reply]
Yeah this was my thinking, black background so fits with all emulator menus.

Would having highlighted text on scroll be possible? Then maybe a toggle to set album art on or off (does this already exist?)

**Reply by Aemiii91** [reply]
The scrolling text is not possible without access to the source. And yes, I'm working on a tweak, which will then have 4 states: Show thumbnails: "same as theme (-)", "gradient", "solid", and "off".. what do you think?

**Reply by sere83** [reply]
Oh yes, that sounds great! Amazing work. Will thin separator line on edge be possible too like in mock-up? Think might help to make separation of artwork and rom list more clear. Gradient or background opacity option is good idea.

**Reply by Aemiii91** [reply]
Yes, for sure.

**Reply by sere83** [reply]
Amazing man, I look forward to seeing it implemented in an upcoming update.

**Reply by sere83** [reply]
> Yes, for sure.

Hey I have OnionOS 4.0.3 installed now. Did this ever get implemented? Couldn't see in the options.

**Reply by Aemiii91** [reply]
Not yet

**Reply by sere83** [reply]
> Not yet

Cool, no worries, hopefully it will make it in soon maybe? ;)

**Reply by Aemiii91** [reply]
Yeah it's part of #571

**Reply by sere83** [reply]
oh lovely, so may make it into 4.1 release?

**Reply by grimhoff** [reply]
Not wanting to raise the dead, but I stumbled upon this thread as I’m also very interested in this UI update.

**Comment by Aemiii91** [comment]
This feature is part of #571
